content/tickets.md

```diff
@@ -16,4 +16,4 @@ those who would otherwise not be able to afford to attend the conference.
 
 Choose your ticket below:
 
-<tito-widget event="pyconuk/2016"></tito-widget>
+<tito-widget event="pyconuk/2016">Click [here](https://ti.to/pyconuk/2016) if you don’t see a ticket form</tito-widget>
```

Comment:
``Visit [our ticket page on Ti.to](https://ti.to/pyconuk/2016) if you don’t see a ticket form``

- no clicks, no heres!